src/javascripts/components/code-blocks.mjs

```diff
@@ -0,0 +1,46 @@
+/**
+ * Responsive behaviour for code blocks
+ */
+class CodeBlocks {
+  /**
+   * @param {NodeListOf<Element>} $elements
+   */
+  constructor($elements) {
+    if (!$elements.length) {
+      return this
+    }
+
+    // Check if we can use Resize Observers, backing out if not
+    if (!('ResizeObserver' in window)) {
+      return this
+    }
+
+    const observer = new window.ResizeObserver((entries) => {
+      for (const entry of entries) {
+        console.log(this.isOverflowing(entry.target), entry)
```

Comment:
Is this console log a carryover from development?

Reply:
Yes it is. Get outta here, you!